muxpi/dut.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+echo "Setting up DUT" 
+#Clear muxpi display and print 'DUT'
+stm_armv7 -clr
+stm_armv7 -print "DUT"
+
+#Make the SD card adapter emulate a real SD-card. Note that the SD card wont be visible on muxpi when its on DUT mode.
+stm_armv7 -dut
+echo "Waiting for the screens..."
+sleep 30s
+
+stty -F /dev/ttyACM0 115200 cs8 -cstopb -parenb
+cd /home/muxpi/
+#We need one screen to save the output of /dev/ttyACM0. Everything will be
+#saved on 'serial-output'
+screen -d -m -S output 
+screen -S output -X stuff 'cat /dev/ttyACM0 > serial-output\n' 
```

Comment:
What if any previous command fails before reaching this command? You don't e.g. rm serial-output at the start so there might be a stale one around... right?

Reply:
Now i always remove it when the script starts